examples/BulkChangeReplyEmail/main.bal

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/io;
+import ballerina/oauth2;
+import ballerinax/hubspot.marketing.emails as hsmemails;
+
+configurable string clientId = ?;
+configurable string clientSecret = ?;
+configurable string refreshToken = ?;
+
+public function main() returns error? {
+
+    // Create the config for authorization to the API
+    hsmemails:OAuth2RefreshTokenGrantConfig auth = {
+        clientId: clientId,
+        clientSecret: clientSecret,
+        refreshToken: refreshToken,
+        credentialBearer: oauth2:POST_BODY_BEARER // this line should be added to create auth object.
```

Comment:
Since the field names and the variables assigned to them are the same, you can use shorthand notation here.

```suggestion
        clientId,
        clientSecret,
        refreshToken,
        credentialBearer: oauth2:POST_BODY_BEARER // this line should be added to create auth object.
```